flink-end-to-end-tests/test-scripts/test_ha_datastream.sh

```diff
@@ -45,7 +45,8 @@ function run_ha_test() {
     # change the pid dir to start log files always from 0, this is important for checks in the
     # jm killing loop
     set_config_key "env.pid.dir" "${TEST_DATA_DIR}"
-    set_config_key "env.java.opts.all" "-ea"
+    set_config_key "env.java.opts.jm" "-ea"
+    set_config_key "env.java.opts.tm" "-ea"
```

Comment:
There are no occurrences for `env.java.opts.jm` and `env.java.opts.tm` in the code. There are `env.java.opts.jobmanager` and `env.java.opts.taskmanager` in `flink-dist/src/main/flink-bin/bin/config.sh`, though. :thinking:

Reply:
oops